src/runtime/distributed/coordinator/kernels/DistributedWrapper.h

```diff
@@ -25,6 +25,8 @@
 #include <runtime/distributed/coordinator/kernels/DistributedCompute.h>
 
 #include <runtime/local/datastructures/AllocationDescriptorGRPC.h>
+#include <runtime/local/datastructures/AllocationDescriptorMPI.h>
+#include <runtime/distributed/worker/MPICoordinator.h>
```

Comment:
Functionality in this file looks pretty generic to me. So these includes should not be necessary? If you want to integrate the coordinator as a worker then you could maybe achieve this by running a worker process on the coordinator node?